### PR TITLE
boards/fvp-armv8r: Update defconfig after running refresh.sh

### PR DESCRIPTION
**Pull request by acassis**
## Summary
Update defconfig after running refresh.sh
## Impact
Only fvp-armv8r
## Testing
N/A

### CODE REVIEW

**Review by xiaoxiang781216 — APPROVED** [review]

**Comment by xiaoxiang781216** [comment]
it's strange that ci doesn't catch this problem.

**Review by jerpelea — APPROVED** [review]
can you please add a commit message

File: boards/arm64/fvp-v8r/fvp-armv8r/configs/nsh/defconfig [hunk 1]
```diff
@@ -5,74 +5,46 @@
...
-CONFIG_ARCH_CHIP_FVP_ARMV8R=y
-CONFIG_ARCH_CHIP_FVP_R82=y
-CONFIG_ARCH_EARLY_PRINT=y
-CONFIG_ARCH_INTERRUPTSTACK=4096
-CONFIG_ARM64_STRING_FUNCTION=y
+# CONFIG_ARCH_FPU is not set
+# CONFIG_NSH_ARGCAT is not set
+# CONFIG_NSH_CMDOPT_HEXDUMP is not set
+CONFIG_ARCH="arm"
+CONFIG_ARCH_BOARD="stm32f4discovery"
```
Comment (raiden00pl) [inline boards/arm64/fvp-v8r/fvp-armv8r/configs/nsh/defconfig:12]: this doesn't look good :)

Comment (acassis) [reply boards/arm64/fvp-v8r/fvp-armv8r/configs/nsh/defconfig:12]: wow! Good finding!

I just run ./tools/refresh.sh and it created these modifications!

**Comment by acassis** [comment]
> it's strange that ci doesn't catch this problem.

Could you please run:
```
./tools/refresh.sh --silent --defaults all
```

And see if same issue file will be updated?

**Comment by acassis** [comment]
I will close this PR, but please guys run the same build script to confirm it is failing. I don't remember if I ran make distclean before running it, probably it was configured to stm32f4discovery.

**Pull request closed without merging**